source/GameAction.cpp

```diff
@@ -40,10 +40,11 @@ namespace {
 	{
 		// Maps are not transferrable; they represent the player's spatial awareness.
 		int mapSize = outfit->Get("map");
+		bool mapMinables = outfit->Get("map minables");
 		if(mapSize > 0)
 		{
```

Comment:
This is only used inside the if statement, so might as well move it into there.
```suggestion
		if(mapSize > 0)
		{
			bool mapMinables = outfit->Get("map minables");
```